packages/interactor/src/constructor.ts

```diff
@@ -100,9 +102,10 @@ function unsafeSyncResolveUnique<E extends Element>(options: InteractorOptions<E
   return resolveUnique(unsafeSyncResolveParent(options), options) as E;
 }
 
-export function instantiateInteractor<E extends Element, F extends Filters<E>, A extends Actions<E>>(
+function createInteractor<E extends Element, F extends Filters<E>, A extends Actions<E>>(
```

Comment:
It's a bit confusing that there are now two functions called `createInteractor` which have different purposes. Can we rename this to something else?

Reply:
I agree. It is a little bit messy with `initInteractor` and `instantiateInteractor`. If I rename `createInteractor` to `instantiateBaseInteractor` will it be ok?